users/nchristus/nchristus.h

```diff
@@ -0,0 +1,75 @@
+#ifndef USERSPACE
+#define USERSPACE
+
+#include "quantum.h"
```

Comment:
A clever trick is to do this:
```suggestion
#include QMK_KEYBOARD_H
```

This also allows you to *just* include this file in your keyboard stuff.

Reply:
Very cool. Updated.